composer.json

```diff
@@ -21,13 +21,13 @@
         }
     ],
     "require": {
-        "prewk/result": "*",
+        "prewk/result": ">= 1.2.0",
```

Comment:
Hm, this is due to PHP version requirements I assume (the variadic release). The original `*` in both packages existed to make sure there would be no dependency hell for people using both packages but `1.2.0` is old enough I guess.

Reply:
Exactly. The tests didn't pass on some job with lowest dependencies where a version of `prewk/result` below `1.2.0` was pulled because of this.

While I'm not a fan of not specifying hard versions constraints for these reasons, you're the maintainer and I didn't wanted to force my style while trying to make things work.